### PR TITLE
Done hw 2

### PR DESCRIPTION
**Pull request by Temporalitas**

### CODE REVIEW

**Comment by klepnevEA** [comment]
Commit

File: src/components/Chat/index.js [hunk 1]
```diff
@@ -1 +1,42 @@
...
+  state = { messages: [], messageInput: `` };
+
+  changeInputMessage(e) {
+    this.setState({ messageInput: e.target.value });
+  }
+
+  sendMessageOnEnter(e) {
+    if (e.key === 'Enter') {
+      this.setState({
+        messages: this.state.messages.concat({ text: this.state.messageInput }),
```
Comment (taksenov) [inline src/components/Chat/index.js:15]: `messages: [...messages, { text: messageInput }]`

File: src/components/Chat/index.js [hunk 2]
```diff
@@ -1 +1,42 @@
...
+import './Chat.css';
+
+class Chat extends React.Component {
+  state = { messages: [], messageInput: `` };
+
+  changeInputMessage(e) {
+    this.setState({ messageInput: e.target.value });
+  }
+
+  sendMessageOnEnter(e) {
```
Comment (taksenov) [inline src/components/Chat/index.js:12]:
```js
  sendMessageOnEnter = event => {
    const { messageInput, messages } = this.state;
```

File: src/components/Chat/index.js [hunk 3]
```diff
@@ -1 +1,42 @@
...
+    if (e.key === 'Enter') {
+      this.setState({
+        messages: this.state.messages.concat({ text: this.state.messageInput }),
+        messageInput: ``
+      });
+    }
+  }
+
+  render() {
```
Comment (taksenov) [inline src/components/Chat/index.js:21]:
```js
  render() {
    const { messageInput, messages } = this.state;
    ...
    <input
          className="input-message"
          onChange={this.changeInputMessage}
          onKeyPress={this.sendMessageOnEnter}
          value={messageInput}
        />
```

**Review by taksenov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**